controllers/rcj_soccer_referee_supervisor/referee/progress_checker.py

```diff
@@ -0,0 +1,44 @@
+import math
+
+
+class ProgressChecker(object):
+    def __init__(self,
+                 steps: int,
+                 threshold: float):
```

Comment:
```python
    def __init__(
        self,
        steps: int,
        threshold: float
    ):
```

Reply:
Cannot say it would look better to me but I guess it is closer to being more "Pythonic" :)

Reply:
Well, seems to be part of an ongoing discussion: https://github.com/python/typing/issues/433

It's quite tough for me to make up my mind as to what actually looks nicer so let's go with this for the time being.